### PR TITLE
Updated Kotlin to 1.4.21

### PR DESCRIPTION
**Pull request by ikurek**
Hello, I'm fairly new to these docsets, but I've noticed that Kotlin docset is pretty old, so I've made an attempt to update it to latest available version. I've tested it and so far it looks like everything works nicely, but additional checks and validations would be appreciated 😄

* Kotlin docset updated to 1.4.21
* Added CSS tweaks to remove static headers, footers and navigation components (should probably resolve [this issue](https://github.com/Kapeli/Dash-User-Contributions/issues/3030))
* Updated Readme file for docset with my data and URL to my version of `kotlin2docset` repo, which contains description of additional CSS fixes

### CODE REVIEW

**Comment by Kapeli** [comment]
Thank you!

Looks great. If you can, please add the following to `Info.plist`: 
```html
<key>DashDocSetDeclaredInStyle</key>
<string>originalName</string>
```

Adding this will remove the file name being displayed next to search results.

File: docsets/Kotlin/docset.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "Kotlin",
-    "version": "1.3.70",
-    "archive": "kotlin.tgz",
+    "version": "1.4.21",
+    "archive": "Kotlin.tgz",
     "author": {
         "name": "Alexander Ronald Altman",
```
Comment (pthariensflame) [inline docsets/Kotlin/docset.json:6]: Whoops; here’s a place my name wasn’t replaced.  >.<

(Sorry for not getting around to this myself.)

Comment (ikurek) [reply docsets/Kotlin/docset.json:6]: I did not intend to remove it. I guess I'm the maintainer now, but I'm not the author, I used an existing script and some tweaks, so I just left `author` unedited 🙂

Comment (pthariensflame) [reply docsets/Kotlin/docset.json:6]: Oh, I see!  I interpreted “author” differently when I initially took over maintainance.  I guess we’ll have to ask @Kapeli which interpretation was intended?  🤷

Comment (Kapeli) [reply docsets/Kotlin/docset.json:6]: I have no idea to be honest. I think the current interpretation for most docsets is "current maintainer".

**Pull request merged**